src/components/PreferencesPage.vue

```diff
@@ -184,83 +184,77 @@
         />
     </label>
     <div v-if="sponsorBlock">
-        <label class="pref" for="chkSkipSponsors">
+        <label class="pref" for="ddlSkipSponsors">
             <strong v-t="'actions.skip_sponsors'" />
-            <input
-                id="chkSkipSponsors"
-                v-model="skipSponsor"
-                class="checkbox"
-                type="checkbox"
-                @change="onChange($event)"
-            />
+            <select id="ddlSkipSponsors" v-model="skipSponsor" class="select w-auto" @change="onChange($event)">
+                <option v-t="'actions.no'" value="no" />
```

Comment:
To keep it easier to overview, maybe we should create an array containing the possible options and then use
```vue
<option v-for="option in sbOptions" v-t=`actions.skip_sponsor_${option}` :value="option">`
```
which would save us some repeating lines and allow extending it in the future (if we wanted to)

Reply:
True, we could potentially even maybe create a component for this.

Reply:
Yeah, thought about that too but I wasn't sure whether we can really make that possible in a good way as `v-model` requires us to be able to access the variable directly which we can't really do when using a component

Reply:
Though Vue might have a way for it I'm not familiar with, but I've never heard of pointers in JS :upside_down_face:

Reply:
I think maybe putting v-for on the <label> element would be smarter, that way we could avoid all that repetition altogether?

Reply:
But how should we use the booleans like `skipIntro` as v-model then?

Reply:
Good question. I'm not familiar with vue at all, but let's see what I can come up with.

Reply:
https://vuejs.org/guide/components/v-model.html seems to be helpful for this.

Reply:
Latest commit should do it.